test/features/can_show_collections_test.rb

```diff
@@ -5,30 +5,85 @@ class CanShowCollectionsTest < Capybara::Rails::TestCase
   include Helpers::UserHelpers
 
   describe 'Showing Collections' do
+
+    describe 'when the user is an Arc Curator' do
+      before do
+        OmniAuth.config.test_mode = true
+        mock_login(role: 'arc_curator')
+      end
+
+      it 'cannot see link for editing mmt' do
+        visit '/home'
+
+        within '#open' do
+          # checks the check box next to the first collection record in the table
+          check 'record_id_'
+
+          within '.navigate-buttons' do
+            click_on 'See Review Detail'
+          end
+        end
+
+        # Tests to see if the edit collection in mmt link is there.
+        page.wont_have_link('EDIT COLLECTION IN MMT')
+      end
+    end
+  end
+
+  describe 'when the user is an Admin' do
     before do
       OmniAuth.config.test_mode = true
-      mock_login(role: 'arc_curator')
+      mock_login(role: 'admin')
+    end
+
+    it 'cannot see link for editing mmt' do
+      visit '/home'
+
+      within '#open' do
+        # checks the check box next to the first collection record in the table
+        check('record_id_')
+
+        within '.navigate-buttons' do
+          click_on 'See Review Detail'
+        end
+      end
+
+      # Tests to see if the edit collection in mmt link is there.
+      page.wont_have_link('EDIT COLLECTION IN MMT')
+    end
+  end
+
+  describe 'when the user is a Daac Curator' do
+    before do
+      OmniAuth.config.test_mode = true
+      # TODO: looking at the uids in user.yml, this user's provider doesn't seem
+      # to match the provider of the collection that shows up in screenshots,
+      # but this is the uid that works.
+      mock_login(role: 'daac_curator', uid: 5)
```

Comment:
So uid:5 is podaac, and doesn't what shows up is record 
```
podaac_in_daac_review:
  id: 14
  recordable_id: 2
  recordable_type: "Collection"
  revision_id: "7"
  state: "in_daac_review"
  format: "dif10"

```
which points to collection.yml #2 which is a podaac record?

Reply:
wait.. record_data.yml I think determines the daac the collection belongs to.   and that is:
```
in_daac_review_short_name:
  id: 20
  record_id: 12
  column_name: "ShortName"
  value: "A2_DySno_NRTb"
  daac: "PODAAC"
  last_updated: null
  color: "green"
  script_comment: "ok"
  opinion: false
  flag: []
  recommendation: "ok"


in_daac_review_long_name:
  id: 21
  record_id: 12
  column_name: "LongName"
  value: "Test_Long_Name"
  daac: "PODAAC"
  last_updated: null
  color: "green"
  script_comment: "ok"
  opinion: false
  flag: []
  recommendation: "ok"


in_daac_review_version_id:
  id: 22
  record_id: 12
  column_name: "VersionId"
  value: "0"
  daac: "PODAAC"
  last_updated: null
  color: "white"
  script_comment: "ok"
  opinion: true
  flag: []
  recommendation: "ok"

```

Reply:
The concept id and short name that shows up (it's not letting post the screenshot) is actually this one from the collections.yml file:
```
first:
  id: 1
  concept_id: "C1000000020-LANCEAMSR2"
  short_name: "A2_DySno_NRT"
```